examples/vision/oml_omniglot.py

```diff
@@ -125,9 +145,18 @@ def main(
         print('Meta Valid Error', meta_valid_error / meta_batch_size)
         print('Meta Valid Accuracy', meta_valid_accuracy / meta_batch_size)
 
-        # Average the accumulated gradients and optimize
-        for p in maml.parameters():
-            p.grad.data.mul_(1.0 / meta_batch_size)
+        # sample remember/random set
+        random_batch = tasksets.train.sample()
+        random_data, random_labels = sample_random_set(
+            random_batch, shots, ways, device
+        )
+        # concatenate with trajectory
+        meta_training_test_data = torch.stack(evaluation_data_all_tasks, random_data)
+        meta_training_test_labels = torch.stack(evaluation_labels_all_tasks, random_labels)
```

Comment:
Concatenates the random set, sampled from all tasks (called remember set in ANML), with samples from the trajectory. Notice the samples are not the same used in inner loop.